backend/apps/matching/configuration/participant.py

```diff
@@ -5,7 +5,22 @@
 class ParticipantConfig:
 
     properties = None
-    signup_layout = None
+    signup_layout_all = None
+    signup_layout_private = None
+
+    def signup_and_edit_layout(self):
+        if self.signup_layout_all is None:
+            self.signup_layout_all = [
+                p.get_signup_layout(ignore_private=True) for p in self.properties
+            ]
+        return self.signup_layout_all
+
+    def view_layout(self):
+        if self.signup_layout_private is None:
+            self.signup_layout_private = [
+                p.get_signup_layout(ignore_private=False) for p in self.properties
+            ]
+        return self.signup_layout_private
```

Comment:
Maybe we can choose a better name? It is not clear what is private here, private like internal? Private like user-private? admin-private? authenticated-private?

Reply:
Well this was intended for this PR 🙈 or at least an explanation in a comment? I myself don't understand what it means 😅

Reply:
@feeds